### ISSUE TITLE
Deprecation warning for svg() function doesn't show template source where the warning occurs

### ISSUE THREAD

**Issue by mdominguez** [craftcms/cms#6927]
### Description
When updating to +3.5, deprecation warnings for the svg() function doesn't show the template source of where this deprecation occurs => [view screenshot](https://share.getcloudapp.com/L1uQxeQQ)

### Steps to reproduce

1. Add a class to an embedded svg: {{ svg('xxx.svg', class='foo') }} (the <3.5 way)
2. View deprecation warnings and notice that the template source file doesn't show

### Additional info

- Craft version: +3.5
- PHP version: 7.4
- Database driver & version: /
- Plugins & versions: /

**Comment by brandonkelly** [comment]
We should be able to fix this, but will need a little more context to reproduce. Please open up the file specified by the deprecation error. At the very top there should be a PHP comment. What does it say?

**Comment by mdominguez** [comment]
No PHP comment to be found in this compiled template?

```
<?php

use Twig\Environment;
use Twig\Error\LoaderError;
use Twig\Error\RuntimeError;
use Twig\Extension\SandboxExtension;
use Twig\Markup;
use Twig\Sandbox\SecurityError;
use Twig\Sandbox\SecurityNotAllowedTagError;
use Twig\Sandbox\SecurityNotAllowedFilterError;
use Twig\Sandbox\SecurityNotAllowedFunctionError;
use Twig\Source;
use Twig\Template;

/* _components/_header */
class __TwigTemplate_47b5d9bc47d940c619df3acca6d5cb6e1473f8aa3f10b9192330d03065a6e60f extends \Twig\Template
{
    private $source;
    private $macros = [];

    public function __construct(Environment $env)
    {
        parent::__construct($env);

        $this->source = $this->getSourceContext();

        $this->parent = false;

        $this->blocks = [
        ];
    }

    protected function doDisplay(array $context, array $blocks = [])
    {
        $macros = $this->macros;
        craft\helpers\Template::beginProfile("template", "_components/_header");
        // line 1
        echo "
";
        // line 5
        echo "
";
        // line 6
        $macros["render"] = $this->macros["render"] = $this->loadTemplate("_macros/_render", "_components/_header", 6)->unwrap();
        // line 7
        echo "
";
        // line 8
        $context["logoTitle"] = (($context["logoTitle"]) ?? ("project-group"));
        // line 9
        echo "
<header class=\"c-header js-c-header";
        // line 10
        echo twig_call_macro($macros["render"], "macro_modifiers", [((nystudio107\seomatic\Node\Expression\EmptyCoalesceExpression::empty(($context["modifiers"] ?? null)) ? null : ($context["modifiers"] ?? null)) ?? (nystudio107\seomatic\Node\Expression\EmptyCoalesceExpression::empty("") ? null : "")), "c-header", false], 10, $context, $this->getSourceContext());
        echo " ";
        echo twig_call_macro($macros["render"], "macro_classes", [((nystudio107\seomatic\Node\Expression\EmptyCoalesceExpression::empty(($context["classes"] ?? null)) ? null : ($context["classes"] ?? null)) ?? (nystudio107\seomatic\Node\Expression\EmptyCoalesceExpression::empty("") ? null : ""))], 10, $context, $this->getSourceContext());
        echo "\">
    
    ";
        // line 13
        echo "    ";
        // line 14
        echo "    ";
        // line 15
        echo "    <div class=\"c-header__row c-header__row--primary pb-8 pt-9\">
        <div class=\"c-container\">
            
            <div class=\"flex justify-center\">
                           
                <div class=\"c-logo inline-block relative text-0\">
                    
                    <a class=\"c-logo__link block\" href=\"";
        // line 22
        echo twig_escape_filter($this->env, (isset($context["siteUrl"]) || array_key_exists("siteUrl", $context) ? $context["siteUrl"] : (function () { throw new RuntimeError('Variable "siteUrl" does not exist.', 22, $this->source); })()), "html", null, true);
        echo "\">
                            
                        ";
        // line 24
        echo $this->extensions['craft\web\twig\Extension']->svgFunction((("@webroot/svgs/" . (isset($context["logoTitle"]) || array_key_exists("logoTitle", $context) ? $context["logoTitle"] : (function () { throw new RuntimeError('Variable "logoTitle" does not exist.', 24, $this->source); })())) . $this->extensions['craft\web\twig\Extension']->attrFilter("__logo.svg", ["class" => "c-logo__svg"])));
        echo "
                        
                        ";
        // line 26
        echo $this->extensions['craft\web\twig\Extension']->svgFunction((("@webroot/svgs/" . (isset($context["logoTitle"]) || array_key_exists("logoTitle", $context) ? $context["logoTitle"] : (function () { throw new RuntimeError('Variable "logoTitle" does not exist.', 26, $this->source); })())) . "__logo.svg"), null, null, "c-svg");
        echo "
                        
                    </a>
                    
                </div>
                
            </div>
            
        </div>
    </div>
    
</header>";
        craft\helpers\Template::endProfile("template", "_components/_header");
    }

    public function getTemplateName()
    {
        return "_components/_header";
    }

    public function isTraitable()
    {
        return false;
    }

    public function getDebugInfo()
    {
        return array (  84 => 26,  79 => 24,  74 => 22,  65 => 15,  63 => 14,  61 => 13,  54 => 10,  51 => 9,  49 => 8,  46 => 7,  44 => 6,  41 => 5,  38 => 1,);
    }

    public function getSourceContext()
    {
        return new Source("
{# ======================== #}
{# HEADER #}
{# ======================== #}

{% import '_macros/_render' as render %}

{% set logoTitle = logoTitle ?? 'project-group' %}

<header class=\"c-header js-c-header{{ render.modifiers(modifiers ??? '', 'c-header', false) }} {{ render.classes(classes ??? '' ) }}\">
    
    {# ============================================ #}
    {# PRIMARY #}
    {# ============================================ #}
    <div class=\"c-header__row c-header__row--primary pb-8 pt-9\">
        <div class=\"c-container\">
            
            <div class=\"flex justify-center\">
                           
                <div class=\"c-logo inline-block relative text-0\">
                    
                    <a class=\"c-logo__link block\" href=\"{{ siteUrl }}\">
                            
                        {{ svg( '@webroot/svgs/' ~ logoTitle ~ '__logo.svg'|attr({'class': 'c-logo__svg'})) }}
                        
                        {{ svg( '@webroot/svgs/' ~ logoTitle ~ '__logo.svg', class='c-svg' ) }}
                        
                    </a>
                    
                </div>
                
            </div>
            
        </div>
    </div>
    
</header>", "_components/_header", "/Users/xxx/Sites/project/templates/_components/_header.twig");
    }
}
```

**Comment by brandonkelly** [comment]
Thanks, was looking for that `/* _components/_header */` comment. Just wanted to get a sense for what type of template it was. I’m able to reproduce on my end and will look into a fix.

**Comment by brandonkelly** [comment]
Alright, got it sorted, and just released Craft 3.5.12.1 with a fix for this.
